CONTRIBUTION.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to PyTado
+
+Thank you for considering contributing to PyTado! This document provides guidelines to help you get started with your contributions. Please follow the instructions below to ensure a smooth contribution process.
+
+## Development Environment
+
+1. **Prepare your development environment**:
+   - Clone the repository: `git clone https://github.com/yourusername/PyTado.git`
+   - Navigate to the project directory: `cd PyTado`
+   - Set up a virtual environment: `python3 -m venv .venv`
+   - Activate the virtual environment:
+     - On macOS/Linux: `source .venv/bin/activate`
+     - On Windows: `.venv\Scripts\activate`
+   - Install the dependencies: `pip install -r requirements.txt` and `pip install -e .`
+
+2. **Install pre-commit hooks** (Reminder to add pre-commit hooks in the future):
+   - Install pre-commit: `pip install pre-commit`
+   - Set up the pre-commit hooks: `pre-commit install`
```

Comment:
Nitpick: what if we write more streamlined instructions, like a simple "copy-paste and be ready to contribute"? I mean, instead of writing multiple steps and a bullet list of items, we can write something similar to this [example](https://github.com/palazzem/econnect-python?tab=readme-ov-file#dev-environment). My suggestion would be:

---
To contribute to this repository, you should first clone your fork and then setup your development environment.
Clone your repository as follows (replace `yourusername` with your GitHub account name):
```bash
git clone https://github.com/yourusername/PyTado.git
cd PyTado
```

Then, to create your development environment and install the project with its dependencies, execute the following commands in your terminal:

```bash
# Create and activate a new virtual environment
python3 -m venv .venv
source .venv/bin/activate  # or .venv\Scripts\activate on Windows

# Upgrade pip and install all projects and their dependencies
pip install --upgrade pip
pip install -r requirements.txt
pip install -e '.[all]'

# Install pre-commit hooks
pre-commit install
```

What do you think? After cloning, you just copy and paste and you have the env set up instead of following a list of steps.

Reply:
Off-topic (for another PR): can't you put all dependencies in your `pyproject.toml`? You can provide production and optional dependencies (like for testings) and use "tags" to decide what to install. Here is an [example](https://github.com/palazzem/econnect-python/blob/main/pyproject.toml#L26-L50).

In your case, `requests` is a dependency, while `responses` is a dev dependency, so you'll list it below `project.optional-dependencies`.

Reply:
Nevermind about the second comment as it looks like you did it in a newer PR! Let me know about the first.

Also, if you agree, we may move these instructions in the README so they are available as soon as contributors land in your repository. Here we'll put just a link to the README.